src/modules/sdlog2/sdlog2.c

```diff
@@ -385,6 +385,8 @@ int sdlog2_main(int argc, char *argv[])
 
 	if (!strncmp(argv[1], "on", 2)) {
 		struct vehicle_command_s cmd;
+
+		memset(&cmd, 0, sizeof(cmd));
```

Comment:
As in further patch in this PR a new field is added to vehicle_command I choose to zeroing all vehicle_command to make it safe but I can just set this field in all vehicle_command.